README.md

```diff
@@ -17,18 +17,29 @@ Sample description
 1. **Install Daytona**: Follow the [Daytona installation guide](https://www.daytona.io/docs/installation/installation/).  
 2. **Create the Workspace**:  
    ```bash  
-   daytona create <SAMPLE_REPO_URL> 
-   ```  
+   daytona create https://github.com/Dishika18/CrisisMonitor-Daytona.git
```

Comment:
Link is wrong, should be https://github.com/daytonaio/sample-django-crisis-monitor

P.S. I realized that I named the repo `crisis-montior`, I'll rename it after we merge

Reply:
Ok so once the repo is renamed, should I update the README with the new name, or is that not necessary?

Reply:
Update the command to `daytona create https://github.com/daytonaio/sample-django-crisis-monitor` now and I'll change the name later.

Reply:
Ok, I have updated the readme.

Reply:
@Dishika18 please set it exactly as `daytona create https://github.com/daytonaio/sample-django-crisis-monitor`